public/media_engagement/worldbank_plr_sms_traffic.js

```diff
@@ -183,6 +191,7 @@ const update = (data) => {
 
     // Set x and y scales
     const x = d3.scaleTime().range([0, Width]);
+    const x_axis_failed_sms = d3.scaleTime().range([0, Width]);
```

Comment:
```suggestion
    const x_axis_failed_sms_range = d3.scaleTime().range([0, Width]);
```
or 
```suggestion
    const x_axis_failed_sms_scale = d3.scaleTime().range([0, Width]);
```
for added clarity